core/pkg/ingress/controller/controller.go

```diff
@@ -944,9 +956,6 @@ func (ic *GenericController) createServers(data []interface{},
 						servers[host].SSLCertificate = cert.PemFileName
 						servers[host].SSLPemChecksum = cert.PemSHA
 					}
-				} else {
-					servers[host].SSLCertificate = defaultPemFileName
-					servers[host].SSLPemChecksum = defaultPemSHA
 				}
```

Comment:
Why this `else` statement was removed? This broke default TLS assignment on `FQDN` without a `secretName`. May we reinclude it? @rikatz @aledbf